src/code/fault.c

```diff
@@ -683,87 +782,115 @@ void Fault_WaitForButtonCombo() {
     }
 }
 
-void Fault_DrawMemDumpPage(const char* title, u32* addr, u32 param_3) {
-    u32* alignedAddr;
+void Fault_DrawMemDumpContents(const char* title, u32 addr, u32 arg2) {
+    u32 alignedAddr = addr;
```

Comment:
There's a number of `uintptr_t` usages that would be appropriate, however it's not currently in the repo so I didn't think to add it yet. I suppose I might as well add them now?

Reply:
Given that we have decided to add them (indeed, we have #1089 open to do much of that), now would probably be a good time, at least for this file. We might want to think about reorganising the headers like MM has done to get them a bit more standard, perhaps? Although that's rather out-of-scope here, it could just go in ultratypes until that's done.

Reply:
Its probably best to typedef them for this file and then if this gets in before 1089 I'll clean it up in my PR since you know better than I do what should and shouldn't be `uintptr_t`